### PR TITLE
Events and Consumer are combined into single binary

### PR DESCRIPTION
**Pull request by aneeshkp**

### CODE REVIEW

File: configs/sa.events.config.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
 		"AlertManagerURL": "http://10.19.110.5:9093/api/v1/alerts",
 		"ResetIndex": false,
 		"Debug": true,
+		"ServiceType" : "event",
```
Comment (leifmadsen) [inline configs/sa.events.config.json]: I think I'd prefer to see these as the plural version. `events` and `metrics`.

File: configs/sa.metrics.config.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
 	"CPUStats": false,
 	"DataCount": -1,
 	"UseSample": false,
+	"ServiceType" : "metric",
```
Comment (leifmadsen) [inline configs/sa.metrics.config.json]: Same as before. I think plural version is more natural here. Or you could just accept both.

File: internal/pkg/metrics/metrics.go [hunk 3]
```diff
@@ -1,9 +1,17 @@
-package main
+package metircs
```
Comment (leifmadsen) [inline internal/pkg/metrics/metrics.go]: Did this actually build?

File: internal/pkg/metrics/metrics.go [hunk 4]
```diff
@@ -119,10 +118,14 @@ func getLoopStater(q chan string, everyCount int) func(count int) {
 		}
 	}
 }
-func main() {
+
+//StartMetrics ... entry point to metrics
+func StartMetrics() {
 	// set flags for parsing options
+	fmt.Printf("Entering maain metrics")
```
Comment (leifmadsen) [inline internal/pkg/metrics/metrics.go]: s/maain/main/

File: internal/pkg/metrics/metrics_test.go [hunk 5]
```diff
@@ -1,14 +1,13 @@
-package main
+package metircs
```
Comment (leifmadsen) [inline internal/pkg/metrics/metrics_test.go]: `package metrics`

File: internal/pkg/metrics/metrics.go (hunk 3 above)
Comment (atyronesmith) [reply internal/pkg/metrics/metrics.go]: What he said...

File: configs/sa.events.config.json (hunk 1 above)
Comment (aneeshkp) [reply configs/sa.events.config.json]: Fixed

File: configs/sa.metrics.config.json (hunk 2 above)
Comment (aneeshkp) [reply configs/sa.metrics.config.json]: Fixed

File: internal/pkg/metrics/metrics.go (hunk 3 above)
Comment (aneeshkp) [reply internal/pkg/metrics/metrics.go]: what do you mean?

File: internal/pkg/metrics/metrics.go (hunk 4 above)
Comment (aneeshkp) [reply internal/pkg/metrics/metrics.go]: removed the debug print statement

File: internal/pkg/metrics/metrics_test.go (hunk 5 above)
Comment (aneeshkp) [reply internal/pkg/metrics/metrics_test.go]: oh man... fixed the typo

Comment (aneeshkp) [reply internal/pkg/metrics/metrics_test.go]: renamed

**Pull request merged**